static_files/css/style.css

```diff
@@ -7,7 +7,34 @@ html {
 }
 
 @import url('https://fonts.googleapis.com/css2?family=Dongle:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Chenla&family=Dongle:wght@300&display=swap');
 
 body{
     font-family: "Dongle", sans-serif;
+    min-height: 100vh;
 }
+
+header{
+    min-height: 50px;
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+}
+
+main{
+    flex: 1;
+}
+
+.Footer {
+    min-height: 50px;
```

Comment:
What do you think about setting the footer's min-height to the value defined in the prototype?

![image](https://user-images.githubusercontent.com/105747728/232605887-be8e217c-9e1c-492b-9442-1f9f914d13a3.png)